app/assets/javascripts/browse-columns.js

```diff
@@ -353,6 +353,9 @@
       this.$breadcrumbs.html($newBreadcrumbs.html());
     },
     trackPageview: function(state){
+      // We want to track eCommerce on both real page views (done via static)
+      // and also these pretend page views (done here via ajax)
+      GOVUK.Ecommerce.start();
```

Comment:
If we don't add this, we only get eCommerce tracking on the first page load. All other AJAX triggered "page views" miss the eCommerce payload.